exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -29,18 +29,32 @@ import (
 	"go.opentelemetry.io/collector/consumer/pdata"
 	"go.uber.org/zap"
 
+	dtMetric "github.com/dynatrace-oss/dynatrace-metric-utils-go/metric"
+	"github.com/dynatrace-oss/dynatrace-metric-utils-go/metric/apiconstants"
+	"github.com/dynatrace-oss/dynatrace-metric-utils-go/metric/dimensions"
 	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dynatraceexporter/config"
-	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dynatraceexporter/serialization"
 )
 
-// The maximum number of metrics that may be sent in a single request to the Dynatrace API
-const maxChunkSize = 1000
-
 // NewExporter exports to a Dynatrace Metrics v2 API
 func newMetricsExporter(params component.ExporterCreateSettings, cfg *config.Config) *exporter {
+	dims := []dimensions.Dimension{}
+	for key, value := range cfg.DefaultDimensions {
+		dims = append(dims, dimensions.NewDimension(key, value))
+	}
+
+	defaultDimensions := dimensions.MergeLists(
+		dimensionsFromTags(cfg.Tags),
+		dimensions.NewNormalizedDimensionList(dims...),
+	)
+	staticDimensions := dimensions.MergeLists(
```

Comment:
I don't think we need to call merge here, as there is only one list here. The normalization is done when calling `NewNormalizedDimensionList`, not in the `Merge` function IIRC.